BRANDING.MD

```diff
@@ -1,37 +1,49 @@
-and it is still early days for the Knative project. For comparison, the
-Kubernetes project did not implement the [Certified Kubernetes Conformance
-Program](https://github.com/cncf/k8s-conformance) until version 1.7. Our goal
-is to have a Conformance Working Group created soon.
+## Knative Brand Guidelines
+
+The Knative trademark and ![Knative Logo](icons/logo.svg "The Knative Logo")
+logo (“Knative Trademarks”) are trademarks of Google, and are treated separately from the copyright or patent license grants contained in the Apache-licensed Knative repositories on GitHub. Any use of the Knative Trademarks other than those permitted in these guidelines must be approved in advance. 
+
+
+
+### 1. Purpose of the Brand Guidelines
+
+These guidelines exist to ensure that the Knative project can share its platform under open source licenses while making sure that the “Knative” brand is protected as a meaningful source identifier in a way that’s consistent with trademark law. 
+
+
+
+### 2. Acceptable Uses
+
+Given the open nature of Knative, you may use the Knative trademark to refer to the project without prior written permission in accordance with these guidelines.
+
+Examples of approved references include the following:
+
+
+*   To refer to the Knative project itself;
+*   To link to knative.dev;
+*   To refer to unmodified source code or other files shared by the Knative repositories on GitHub;
+*   In blog posts, news articles, or educational materials about Knative;
+*   To accurately identify that your design or implementation is based on or is for use with the Knative platform.
+
+Suggested templates: 
+
+*   “[Your Product] for Knative.”
+*   “[XYZ] Conference for Knative Users.”
+
+### 3. Usage Guidelines
+
+*   The Knative name may not be used or registered in a manner that would cause confusion as to Google’s affiliation or endorsement (as the trademark owner).
+*   Don’t use the Knative name as part of your company name, product name, domain name, or social media profile.
+*   Other than as permitted by these guidelines, the Knative name should not be combined with other trademarks, terms, or source identifiers.
+*   Don't remove, distort or alter any element of the Knative Trademarks without permission:
+    *   Do not shorten, abbreviate, or change the spelling of the Knative name, for example in mass communication, product names, social media profiles, domain names or event names).
+    *   Do not change the color or design elements of the Knative logo, or combine the logo with any other text.
+*   User groups, social media accounts, community websites, or other community-organized properties or organizations should follow the naming convention "[XYZ] for Knative Users" or "[XYZ] for Knative Contributors". 
+*   The name "Knative Working Group" can only be used by approved working groups maintaining code in the Knative GitHub organization under Knative Project governance.
```

Comment:
s/organization/organizations/  (add 's') because I think we have more than one.

Reply:
Are you thinking about knative-sandbox @duglin ?

Reply:
Yep; that's a separate GitHub organization. We might have more in the future (though they are a pain to manage when split up, so I hope not too many).

Reply:
yup!